pom.xml

```diff
@@ -165,6 +165,11 @@ under the License.
       <artifactId>maven-reporting-impl</artifactId>
       <version>3.2.0</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-text</artifactId>
+      <version>1.10.0</version>
+    </dependency>
```

Comment:
for transitive version override I like to use dependency Management, it is more clear that we only manage of version but it is not a direct needed dependency.

Reply:
thx - updated